Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java

```diff
@@ -1635,6 +1636,14 @@ private void checkColors(Card card, MtgJsonCard ref) {
         }
     }
 
+    private void checkCardCanBeCopied(Card card) {
+        try {
+            Card card2 = card.copy();
+        } catch (StackOverflowError err) {
+            fail(card, "copy", "throws a StackOverflow on copy.");
```

Comment:
You can catch any exception here, so it can help to find another potential bugs with copy code too